### DISCUSSION TITLE
浅谈大学生参选人大代表

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#26] [Announcements]
# 浅谈大学生参选人大代表

如果你是一名大学生，且渴望参政议政，为同学们做点实事，我希望本文能为你提供些微帮助。我们大学的政治课应当是一门实践课，而不是纸上谈兵。既然政治课不教，我们自学！

https://zexwoo.blog/posts/editorial/democracy/

**Comment by PureDimension** [comment]
感谢博主，等下一次换届感觉可以去玩一玩，亲身感受一下w

**Reply by ZexWoo** [reply]
加油！
